examples/go-pipeline/README.md

```diff
@@ -0,0 +1,32 @@
+# SSF Golang Tekton Pipeline
+
+This is a sample GOlang tekton pipeline.
+
+> :warning: This pipeline is not intended to be used in production
+
+## Starting Demo
+
+```bash
+# Only if a cluster is needed.
+make setup-minikube
+
+# Setup tekton w/ chains
+make setup-tekton-chains tekton-generate-keys setup-kyverno
+
+# Run a new pipeline.
+make example-golang-pipeline
+
+# Export the value of imageUrl from the pipelinerun describe as DOCKER_IMG:
+export IMAGE_URL=$(tkn pr describe --last -o jsonpath='{.spec.params[?(@.name=="image")].value}')
```

Comment:
Could probably eliminate `IMAGE_URL` for this example since `IMAGE_URL==DOCKER_IMG`

Reply:
Agreed